src/trg-main-window.c

```diff
@@ -1514,9 +1514,15 @@ trg_torrent_tree_view_visible_func(GtkTreeModel * model,
             gchar *filterCmp = g_utf8_casefold(filterText, -1);
             gchar *nameCmp = g_utf8_casefold(name, -1);
 
-            if (!strstr(nameCmp, filterCmp))
-                visible = FALSE;
+            GRegex *regex;
+            GMatchInfo *match_info;
+            regex = g_regex_new (filterCmp, 0, 0, NULL);
```

Comment:
Compiling regex every single call is a bit overkill considering the text wouldn't have changed.

Reply:
where should the regex be compiled?